tests/lib/ocpp/v201/test_smart_charging_handler.cpp

```diff
@@ -142,6 +142,25 @@ class ChargepointTestFixtureV201 : public testing::Test {
         };
     }
 
+    ChargingProfile create_charging_station_max_profile(
+        ChargingSchedule charging_schedule, std::string transaction_id, int stack_level = 1,
+        ChargingProfileKindEnum charging_profile_kind = ChargingProfileKindEnum::Absolute) {
+        auto charging_profile_id = 1;
+        auto charging_profile_purpose = ChargingProfilePurposeEnum::ChargingStationMaxProfile;
+        auto recurrency_kind = RecurrencyKindEnum::Daily;
+        std::vector<ChargingSchedule> charging_schedules = {charging_schedule};
+        return ChargingProfile{.id = charging_profile_id,
+                               .stackLevel = stack_level,
+                               .chargingProfilePurpose = charging_profile_purpose,
+                               .chargingProfileKind = charging_profile_kind,
+                               .chargingSchedule = charging_schedules,
+                               .customData = {},
+                               .recurrencyKind = recurrency_kind,
+                               .validFrom = {},
+                               .validTo = {},
+                               .transactionId = transaction_id};
```

Comment:
Why not use `create_charging_profile ()` instead of a new function? Alternatively, this could be made smaller by using `create_charging_profile ()` inside of it to reduce code duplication.

Reply:
I generally lean towards not drying things too much in the tests, but it's not a big deal.

Reply:
But, now that I do it, I think it's a lot cleaner.